bin/init-database

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+const db = require('../db');
+
+const createBadgesTable = async () => {
+  const query = "CREATE TABLE `badges` ( \
+    `id` bigint(20) NOT NULL AUTO_INCREMENT, \
+    `name` varchar(255) NOT NULL, \
+    `description` text, \
+    PRIMARY KEY (`id`) \
+  );";
+
+  try {
+    const connection = await db.connect();
+    await db.query(connection, query);
+    await db.close(connection);
+  } catch (error) {
+    throw error;
+  }
+};
```

Comment:
itt es a tobbinel nem latom ertelmet a try catch hasznalatara, a hiba csak tovabb van dobva, amit amugy is elkapna a Promise catch a promis chain vegen